README.md

```diff
@@ -16,54 +16,104 @@ compiled correctly.
 The library is known to compile on various Linux versions (Redhat,
 Mageia, Ubuntu, Debian), Mac OS X, and Windows (Mingw-w64).
 
-Step 1: Installing LLVM
-=======================
 
```

Comment:
Aha! This ordering problem occurs when I compile my files individually as .bc files, link with llvm-link and then feed the final .bc file into llvm-cbe. 

It looks like the ordering of global variable definitions is lost in the llvm-link stage.